packages/lesswrong/server/search/elastic/elasticCallbacks.ts

```diff
@@ -25,8 +26,8 @@ export const elasticSyncDocument = (
 if (isElasticEnabled) {
   for (const collectionName of searchIndexedCollectionNames) {
     const callback = ({_id}: DbObject) => elasticSyncDocument(collectionName, _id);
-    getCollectionHooks(collectionName).createAfter.add(callback);
-    getCollectionHooks(collectionName).updateAfter.add(callback);
+    getCollectionHooks(collectionName).createAsync.add(({ document }) => callback(document));
+    getCollectionHooks(collectionName).editAsync.add(callback);
```

Comment:
I changed these to async because there is currently an error on dev (for my user at least) that causes elastic updates to fail, which was causing any update to the user to fail. I'll look into that separately, but I don't think it should block updates to the user

Reply:
Reading this again I can even less understand why this was causing updates to fail before, when the update was already in a try/catch block 🤔